src/test/java/org/openmrs/contrib/qaframework/page/AttachmentsPage.java

```diff
@@ -0,0 +1,59 @@
+package org.openmrs.contrib.qaframework.page;
+
+import java.util.Scanner;
+
+import org.openmrs.reference.page.ClinicianFacingPatientDashboardPage;
+import org.openmrs.uitestframework.page.Page;
+import org.openqa.selenium.By;
+
+public class AttachmentsPage extends Page{
+	
+	private static final By ATTACHMENTS_LINK = By.className("icon-share-alt edit-action right");
+	private static final By ATTACH_FILE = By.className("dz-default dz-message ng-binding");
+	private static final By ADD_CAPTION = By.className("ng-pristine ng-valid ng-empty ng-touched");
+	private static final By UPLOAD_FILE = By.className("confirm ng-binding");
+	private static final By CLEAR_FORMS = By.className("ng-binding");
+	public static final By ICON_CAMERA = By.className("icon-camera");
+	public static final By SAVE_BUTTON = By.className("icon-ok");
+	private Scanner userInput = new Scanner(System.in);
+	
+	public AttachmentsPage(ClinicianFacingPatientDashboardPage clinicianFacingPatientDashboardPage) {
+		super(clinicianFacingPatientDashboardPage);
+	}
+
+	@Override
+	public String getPageUrl() {
+		return "/attachments/attachments.page";
+	}
+
+	public void clickOnAttachmentLink() {
+		driver.findElement(ATTACHMENTS_LINK).click();
+	}
+
+	public void clickOrDropFile() {
+		driver.findElement(ATTACH_FILE).click();
+	}
+
+	public String addCaption() {
+		String captionText=userInput.next();
```

Comment:
is there no better way to read the caption from the dom using selenium?

Reply:
Applied sendKeys() to read the user input